command/connect/envoy/testdata/CONSUL_HTTP_ADDR-with-https-scheme-enables-tls.golden

```diff
@@ -190,3 +190,4 @@
     }
   }
 }
+
```

Comment:
This is a side effect of the internal plumbing for `ui.Output()` which seemed fine since trailing whitespace isn't semantically relevant for json.